frontend/src/App.tsx

```diff
@@ -670,12 +675,8 @@ function AppSidebar(props: { routes: (IRoute | IRouteGroup)[] }) {
                                         id="toggle-id"
                                         onToggle={onToggle}
                                         className={classes.perspective}
-                                        icon={
-                                            <span style={{ fill: 'currentColor' }}>
-                                                <ACMPerspectiveIcon />
-                                            </span>
-                                        }
-                                        style={{ fontSize: 'small' }}
+                                        icon={<ACMPerspectiveIcon />}
+                                        style={{ fontSize: 'small', backgroundColor: 'transparent' }}
```

Comment:
Seems not correct for light mode
<img width="335" alt="image" src="https://user-images.githubusercontent.com/42188127/172472942-f3f8f04c-5275-43a1-aa03-375c99feca9b.png">